.circleci/config.yml

```diff
@@ -0,0 +1,76 @@
+version: 2.1
+jobs:
+  build:
+    working_directory: ~/comit
+    machine:
+      image: ubuntu-1604:201903-01
+      docker_layer_caching: true
+    steps:
+      - checkout
+      - restore_cache:
+          key: dependency-cache-{{ checksum "Cargo.lock" }}
+      - setup_system
+      - setup_nodejs
+      - setup_rust
+      - print_current_versions
+      - run:
+          name: Build using cargo make
+          command: |
+            source $HOME/.cargo/env
+            cargo --version
+            cargo make ci
```

Comment:
:thinking: I think we are using `cargo make travis` in the travis file? That one runs `ci-flow` where you are only running `ci`. I think there might be some `pre-*` steps missing then.

Reply:
:x: I saw you deleted the `sccache` setup? This will massively slow down our build times on travis. As long as that one is still our primary build pipeline (PR check, slack notifications) that one should continue as it is IMO.

Why don't you use sccache on circle-ci?

Reply:
❌ please use `cargo make travis` (happy to rename it) so it sets the threads and setup for sscache:
`env = { "RUST_TEST_THREADS" = "8", "RUSTC_WRAPPER" = "${HOME}/.cargo/bin/sccache"  }`

Reply:
Readded sccache